python/src/uagents/agent.py

```diff
@@ -467,7 +505,7 @@ async def register(self):
             or list(self.protocols.keys())
             != self._almanac_contract.get_protocols(self.address)
         ):
-            agent_balance = self._ledger.query_bank_balance(
+            agent_balance = self.ledger.query_bank_balance(
```

Comment:
Can now be updated to `self.balance` right?

Reply:
Thats right